server/lib/sync-channel.ts

```diff
@@ -9,6 +9,8 @@ import { VideoChannelSyncState, VideoPrivacy } from '@shared/models'
 import { CreateJobArgument, JobQueue } from './job-queue'
 import { ServerConfigManager } from './server-config-manager'
 
+const lTags = loggerTagsFactory('youtube-dl')
```

Comment:
Prefer to use `channel-sync` tag, it's not really a `youtube-dl` package even if it uses this module

Reply:
![0fc](https://user-images.githubusercontent.com/981494/195122056-32c12c33-0b5b-4886-9177-7b0953d8e6b2.gif)

Reply:
Nah I get the point. Yeah, will modify.

Reply:
Modified to `channel-sync` tag